Plugins/Flow.Launcher.Plugin.PluginsManager/PluginsManager.cs

```diff
@@ -300,6 +324,7 @@ private void Uninstall(PluginMetadata plugin)
             using var _ = File.CreateText(Path.Combine(plugin.PluginDirectory, "NeedDelete.txt"));
         }
 
+
```

Comment:
how come some times your changes contain extra space?

Reply:
hmmm, I don't know…… Maybe due to auto formatting?